examples/sound/Sound.cpp

```diff
@@ -46,15 +47,15 @@ void playSound()
 /// Play a music
 ///
 ////////////////////////////////////////////////////////////
-void playMusic()
+void playMusic(const sf::String& filename)
```

Comment:
```sf::Music::openFromFile``` takes a ```std::string```, so here you're just giving the false impression that ```sf::String``` (and thus Unicode) is supported :p